### PR TITLE
DocumentationSyntax Docs

### PR DESCRIPTION
**Pull request by i-walker**

### CODE REVIEW

File: idea-plugin/src/main/kotlin/arrow/meta/ide/dsl/editor/documentation/DocumentationSyntax.kt [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+   *          }
+   *         }
+   *        }
+   *       }
+   *      )
+   *     )
+   *    }
+   * //sampleEnd
+   * ```
+   * This example render's quickDocs on local declarations, but is insufficient for toplevel KtDeclarations.
```
Comment (jansorg) [inline idea-plugin/src/main/kotlin/arrow/meta/ide/dsl/editor/documentation/DocumentationSyntax.kt]: nit: render's -> renders

File: idea-plugin/src/main/kotlin/arrow/meta/ide/dsl/editor/documentation/DocumentationSyntax.kt [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+   * In addition, this implementation should not render the complete Doc's only the minimal amount of data.
```
Comment (jansorg) [inline idea-plugin/src/main/kotlin/arrow/meta/ide/dsl/editor/documentation/DocumentationSyntax.kt]: nit: Doc's -> docs?

File: idea-plugin/src/main/kotlin/arrow/meta/ide/dsl/editor/documentation/DocumentationSyntax.kt (hunk 1 above)
Comment (i-walker) [reply idea-plugin/src/main/kotlin/arrow/meta/ide/dsl/editor/documentation/DocumentationSyntax.kt]: makes sense

File: idea-plugin/src/main/kotlin/arrow/meta/ide/dsl/editor/documentation/DocumentationSyntax.kt (hunk 2 above)
Comment (i-walker) [reply idea-plugin/src/main/kotlin/arrow/meta/ide/dsl/editor/documentation/DocumentationSyntax.kt]: thx

**Pull request merged**